actionmailer/test/message_delivery_test.rb

```diff
@@ -76,7 +76,7 @@ def test_should_enqueue_and_run_correctly_in_activejob
 
   test "should enqueue a delivery with a delay" do
     travel_to Time.new(2004, 11, 24, 01, 04, 44) do
-      assert_performed_with(job: ActionMailer::DeliveryJob, at: Time.current.to_f + 600.seconds, args: ["DelayedMailer", "test_message", "deliver_now", 1, 2, 3]) do
+      assert_performed_with(job: ActionMailer::DeliveryJob, at: Time.current.to_f + 600, args: ["DelayedMailer", "test_message", "deliver_now", 1, 2, 3]) do
```

Comment:
Cools. I would even go with Time.current + 10.minutes. That's the wonder of these aliases. You get to pick the scale that makes the most sense.